04-onion-routing.md

```diff
@@ -211,6 +211,10 @@ This is formatted according to the Type-Length-Value format defined in [BOLT #1]
     1. type: 18 (`total_amount_msat`)
     2. data:
         * [`tu64`:`total_msat`]
+    1. type: 20 (`attributable_error`)
+    2. data:
+        * [`byte`:`hop_count`]
+        * [`byte`:`fixed_payload_len`]
```

Comment:
Currently under discussion is the proposal to just remove these fields and use a fixed structure:
* 20 hops max
* 4 byte payload

Combined with hmacs truncated to 4 bytes, this results in a total failure message size of 1200 bytes. This stays below the typical tcp mtu of 1500 bytes.